databricks/koalas/indexes.py

```diff
@@ -456,6 +457,47 @@ def to_series(self, name: Union[str, Tuple[str, ...]] = None) -> Series:
                                          column_index_names=None),
                       anchor=kdf)
 
+    def duplicated(self, keep=False):
```

Comment:
Let's explicitly don't support this since it returns an array which is supposed to be big. Users can easily workaround by `DataFrame.duplicated()` API:

```python
>>> kdf
   id  col
0   0    2
1   1    2
2   2    2
3   3    2
4   4    2
5   5    2
6   6    2
7   7    2
8   8    2
9   9    2
>>> kdf[['col']].duplicated()
0    False
1     True
2     True
3     True
4     True
5     True
6     True
7     True
8     True
9     True
Name: 0, dtype: bool
```

We can include this in the error message.